src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -16,26 +16,26 @@ public class Exercise {
 
     // 2. There are currently some upper case characters in the URL. Using an appropriate string method on the fixedUrl member above,
     // set the value of lowerCasedUrl.
-    public String lowerCasedUrl = "";
+    public String lowerCasedUrl = this.fixedUrl.toLowerCase();
 
 
     // 3. There is still white space on both ends of the URL! Use the appropriate string method to trim that white space
     // and set the value of the url member below
-    public String url = "";
+    public String url = this.lowerCasedUrl.trim();
 
 
     // 4. Using the appropriate string method on url, set the value of the protocol member below
-    public String protocol = "";
+    public String protocol = this.url.substring(0,5);
 
 
     // 5. Using the appropriate string method on url, set the value of the domain member below
-    public String domain = "";
+    public String domain = this.url.substring(8,21);
 
 
     // 6. Set the length member below to the length of the url member
-    public int length = 0;
+    public int length = this.url.length();
 
 
     // 7. Using concatenation and existing members, set the faqUrl member below to the faq page of the boolean website
-    public String faqUrl = "";
+    public String faqUrl = this.url.replaceAll("who-we-are","faq");
```

Comment:
Works, but looking for use of concatenation as per comment. Also, you don't need any of the `this` references here. 
The `this` keyword is used to refer to the current object instance within a method or a constructor, primarily to resolve ambiguity between class fields and parameters or local variables with the same name, or to call other constructors in the same class.